src/xlsx/functions/cells.ts

```diff
@@ -63,18 +63,19 @@ export function addFormula(cell: ExcelCellData): {
 
 export function addContent(
   content: string,
-  sharedStrings: string[]
+  sharedStrings: string[],
+  forceString = false
```

Comment:
table header's content is already forced to strings in `exportForExcel`, right ?

Reply:
Well `cell.content` is always a string, but `addContent` use our `isNumber` helper that use regex to know if the content is a number or a string. 
Using `cell.value` doesn't work, because Error cells then will export their value (error message) instead of their content (formula in error). Maybe we can fix this (adding a cell.isError or setting cell.isFormula to true for the error cells), but I'm not sure it's better